### ISSUE TITLE
Unable to connect EVTV ESP32 Board to SavvyCAN after flashing with ESP32RET

### ISSUE THREAD

**Issue by cypht** [collin80/ESP32RET#27]
I have successfully flashed the EVTV ESP32 board with the ESP32RET Firmware Updater. But when I add a new connection to SavvyCAN, using the GVRET option, it is never able to establish a connection. 

I tried re flashing multiple times and also tried on Windows and MacOS, but same issue.

Please check screenshot, and help me with what I am missing.

<img width="1084" alt="image" src="https://github.com/collin80/ESP32RET/assets/137227341/19ee149e-e9a2-49fe-aeaf-702abc4392a4">

**Comment by collin80** [comment]
I'm noticing that the port listed is a serial port but the debugging output shows it is trying to use an TCP connection. That won't work. Make sure you use the "Serial Connection (GVRET)" option when connecting.

**Comment by cypht** [comment]
I am using the Serial GVERT Connection. But it is still showing a TCP
connection. Not sure why is it doing that. Is there anything I need to
change while flashing the module? Or some other setting in SavvyCAN?

**Comment by collin80** [comment]
No, it should work the way you're doing it. Oh! Actually, I do remember something about this. Windows shouldn't have this behavior but MacOS can trigger the TCP thing because of the dot in the port name. On Windows it'll be named something like COM7 and that doesn't trip up the code. I thought that this was fixed in the latest SavvyCAN but I see that the fix is only if the device name has a . and tty in the name. Yours does not have tty in the name so it thinks it is a domain name to look up. I'll fix this right now.

**Comment by collin80** [comment]
Commit 1999da88c5d6b32bd432b737ba1b60d1a15ff47b fixes the TCP bug on MacOS that you are experiencing. A new CI build should be available momentarily.

**Comment by cypht** [comment]
Thank you for your quick update. I managed to get it to work with tty.xx but not with the prebuilt updater. Manually compiling this project and flashing it worked after selecting tty.xx with GVRET, not with cu.xx, which i believe is specific to MacOS. Once the new build is available i'll retry flashing it and checking again with both. But the prebuilt updater would not be updated right away right?

**Comment by cypht** [comment]
There is another thing that I noticed, the EVTV device keeps disconnecting every 10-15 secs while capturing frames from a vehicle. Is this an OS related bug or something else? Not sure how I can pull logs for the same.

**Comment by collin80** [comment]
It most certainly should not be doing that. However, I just found that the firmware updater was actually very old and was not the proper version that was compiled on Aug 2nd of 2023. So, maybe try redownloading the firmware updater and running it again. I just had someone else running into bad issues and found the problem. Sorry about that!
